plugins/compress/configuration.cc

```diff
@@ -270,16 +270,22 @@ HostConfiguration::compression_algorithms()
   return compression_algorithms_;
 }
 
+/**
+  "true" and "false" are compatibility with old version, will be removed
+ */
 void
 HostConfiguration::set_range_request(const std::string &token)
 {
-  // "true" and "false" are compatibility with old version, will be removed
-  if (token == "false" || token == "ignore-range") {
-    range_request_ctl_ = RangeRequestCtrl::IGNORE_RANGE;
-  } else if (token == "true" || token == "none") {
+  if (token == "true" || token == "none") {
     range_request_ctl_ = RangeRequestCtrl::NONE;
-  } else if (token == "no-compression") {
+  } else if (token == "false" || token == "no-compression") {
```

Comment:
Should you still support `ignore-range` for compatibility?

Reply:
It's introduced by https://github.com/apache/trafficserver/pull/11975 recently and not released yet. ( only master and 10.1.x branch have it ). So no compatibility issue yet.